smileidentity/ios
# Add allowUserToSkipBackCapture parameter

Story: https://app.shortcut.com/smileid/story/11625

## Summary

Adds a new `allowUserToSkipBackCapture` boolean parameter to allow partners to enforce capturing the back of ID. This parameter is only relevant when `captureBothSides` is set to `true`

(see also: https://github.com/smileidentity/android/pull/303)

## Test Instructions

Update HomeView to pass `captureBothSides` to `true` and `allowUserToSkipBackCapture` to `false` for `documentVerificationScreen` and `enhancedDocumentVerificationScreen`. Notice that the skip button doesn't show
